packages/integrations/stores/azure-key-vault/src/azure-key-vault.ts

```diff
@@ -0,0 +1,36 @@
+import { DefaultAzureCredential, type DefaultAzureCredentialClientIdOptions } from '@azure/identity';
+import { SecretClient, type SecretClientOptions } from '@azure/keyvault-secrets';
+import { KeyValueConfigStore } from '@configu/ts';
```

Comment:
Please use the `KeyValueConfigStore` from within the intergations package.
```suggestion
import { KeyValueConfigStore } from '@configu/integrations/src/utils/KeyValue';
```

Reply:
Done